<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MasterLog</class>
 <widget class="QMainWindow" name="MasterLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demented - Master Log</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>User</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Catalog Number</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5" rowspan="5">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="label_Welcome">
      <property name="text">
       <string>Welcome, </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>From</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateTimeEdit" name="dateTimeEdit_from"/>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="comboBox_User"/>
    </item>
    <item row="2" column="3" colspan="2">
     <widget class="QLineEdit" name="lineEdit_CatNum"/>
    </item>
    <item row="3" column="2" rowspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Action</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" rowspan="2">
     <widget class="QDateTimeEdit" name="dateTimeEdit_to"/>
    </item>
    <item row="4" column="6" rowspan="2">
     <widget class="QPushButton" name="pushButton_Logout">
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </item>
    <item row="4" column="7" rowspan="2">
     <widget class="QPushButton" name="pushButton_Menu">
      <property name="text">
       <string>Menu</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>To</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QComboBox" name="comboBox_Action"/>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="pushButton_Search">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="pushButton_Clear">
      <property name="text">
       <string>Clear Results</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="8">
     <widget class="QTableView" name="tableView_Log"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_CatNum</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton_Search</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
